src/app/header/header.component.ts

```diff
@@ -1,15 +1,20 @@
 import { Component, OnInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
-
-  constructor() { }
-
+  id = 0;
+  lang = ["Changer en Français", "Change to English"];
```

Comment:
why is this here?
you can get the value directly in the template by calling translateService.currentLang